project/ProjectAutoPlugin.scala

```diff
@@ -35,6 +35,7 @@ object ProjectAutoPlugin extends AutoPlugin {
     crossScalaVersions := Dependencies.ScalaVersions,
     scalaVersion := Dependencies.Scala212,
     Test / fork := true,
+    Test / javaOptions ++= Seq("-Xms64m", "-Xmx128m"),
```

Comment:
This is not optimal. I want to make it different. 

My plan is to exclude the test from the normal run and have another job that runs only this memory test. 

We will need an sbt task that tweaks the `javaOptions` for it. Passing `-Xms64m -Xmx128m` is not good as it is not passed over to forked tests.